src/fields/commandLine.ts

```diff
@@ -0,0 +1,19 @@
+import type { Field } from 'payload/types'
+
+const commandLine: Field = {
+  name: 'commandLine',
+  type: 'group',
```

Comment:
Is there a reason we opted for a group here? Are we thinking of possibly adding more fields to it in the future?